Makefile

```diff
@@ -35,6 +35,49 @@ export VALIDATE_REPO
 export VALIDATE_BRANCH
 export VALIDATE_ORIGIN_BRANCH
 
+ifneq "$(strip $(shell command -v go 2>/dev/null))" ""
```

Comment:
This all seems a bit overkill.
What are we trying to solve here?

Reply:
@cpuguy83 I ran into this problem;

```
 /bin/sh: 4: go: not found
 /bin/sh: 5: go: not found
```

Reply:
Likely because the `GOOS` and `GOARCH` make variables were not set

Reply:
Yeah but doesn't what we already have solve that? And in the case no `go` the user just needs to set `GOOS` and `GOARCH` themselves.

Reply:
But I just discovered what the actual problem was; we were now actually using buildx v0.3.0, so it tried to pull an image that doesn't exist for s390x/power. The `go not found` was (I think) just the output leaking from the `go env GOHOSTOS || uname` line; I'll remove that one commit

Reply:
Oh, actually.. (sleepy head); it's because s390x/power are now using buildkit to build the image 🤔